### PR TITLE
config babel-plugin-remove-console with eslint no-console and the fixes

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**

### CODE REVIEW

File: libs/components/src/RadioGroup/RadioGroup.js [hunk 1]
```diff
@@ -30,7 +30,13 @@ const defaultRenderOption = ({
       onChange: handleChange,
       value: option.value,
     }),
-    React.createElement('label', { htmlFor: id }, option.label)
+    React.createElement(
```
Comment (eckdanny-osi) [inline libs/components/src/RadioGroup/RadioGroup.js]: The original source must be TypeScript. We don't typically invoke `createElement`

Comment (NagaPradeepGajula-osi) [reply libs/components/src/RadioGroup/RadioGroup.js]: Haven't changed anything here it was prettified on save.

File: package.json [hunk 2]
```diff
@@ -51,6 +51,7 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-rename-import": "^2.3.0",
+    "babel-plugin-transform-remove-console": "6.9.4",
```
Comment (eckdanny-osi) [inline package.json]: if we want to rake console API from dist artifacts, we'll need to configure rollup for the lib projects. Opening an issue to close this.

Comment (NagaPradeepGajula-osi) [reply package.json]: 👍

**Pull request merged**